src/apis/avantation.ts

```diff
@@ -49,8 +49,10 @@ export class AvantationAPI implements Avantation.InputConfig {
         this.run();
     }
 
-    private async run() {
-        this.har.log.entries.forEach(this.buildEntry.bind(this));
+    private run() {
```

Comment:
Having `run` as an `async` method in the constructor was making this difficult to test - does it need to be async?